src/App.js

```diff
@@ -102,17 +101,7 @@ class App extends Component {
         <header className="App-header">
           {this.state.user ? <HeaderNav user={this.state.user} /> : null}
           <img src={logo} className="App-logo" alt="logo" />
-          <p>
```

Comment:
Why did you remove this internalization example from the main page?

Reply:
i update this pr fixing this